lib/api/model/pokemon_move.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'pokemon_move.freezed.dart';
+part 'pokemon_move.g.dart';
+
+@freezed
+class PokemonMove with _$PokemonMove {
+  factory PokemonMove({
+    @JsonKey(name: 'name') required String name,
+    @JsonKey(name: 'url') required String url,
+  }) = _PokemonMove;
+
+  factory PokemonMove.fromJson(Map<String, dynamic> json) =>
+      _$PokemonMoveFromJson(json);
```

Comment:
make this one liner as well

Reply:
Hello Sir, it comes outside the vertical line limit. Should I remove the limit or adjust it?

Reply:
ayt, keep it as is then 👍

Reply:
marking as resolved 🙂